neuralmonkey/run.py

```diff
@@ -127,10 +127,16 @@ def main() -> None:
 
             dataset = dataset.subset(start, length)
 
+
+        if CONFIG.model.runners_batch_size is None:
+            runners_batch_size = CONFIG.model.batch_size
```

Comment:
nedaval bych, uplne stejne se nedavaji pri trenovani, bral bych to spis jako specialni moznost navic definovat runners batch (osobne to treba ve vetsine experimentu nedefinuju)